content/en/agent/cluster_agent/setup.md

```diff
@@ -45,153 +66,121 @@ If you are using Azure Kubernetes Service (AKS), you may require extra permissio
 
 ### Secure Cluster Agent to Agent communication
 
-Use one of the following options to secure communication between the Datadog Agent and the Datadog Cluster Agent.
-
-* Create a secret and access it with an environment variable.
-* Set a token in an environment variable.
-* Use a ConfigMap to manage your secrets.
-
-Setting the value without a secret results in the token being readable in the `PodSpec`.
-
-{{< tabs >}}
-{{% tab "Secret" %}}
-
-1. Run the following command to create a secret token. Your token must be at least 32 characters long.
-
-    ```shell
-    echo -n '<ThirtyX2XcharactersXlongXtoken>' | base64
-    ```
-
-2. Run this one line command:
-
-    ```shell
-    kubectl create secret generic datadog-cluster-agent --from-literal=token='<ThirtyX2XcharactersXlongXtoken>'
-    ```
-
-    Alternatively, modify the value of the secret in the `agent-secret.yaml` file located in the [manifest/cluster-agent directory][1] or create it with:
-
-    `kubectl create -f Dockerfiles/manifests/cluster-agent/agent-secret.yaml`
-
-3. Refer to this secret with the environment variable `DD_CLUSTER_AGENT_AUTH_TOKEN` in the manifests of the Cluster Agent. See [Create the Cluster Agent and its service][2] and [Configure the Datadog Cluster Agent][3].
-
-[1]: https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/manifests/cluster-agent/agent-secret.yaml
-[2]: /agent/cluster_agent/setup/?tab=secret#create-the-cluster-agent-and-its-service
-[3]: /agent/cluster_agent/setup/?tab=secret#configure-the-datadog-agent
-{{% /tab %}}
-{{% tab "Environment Variable" %}}
+The Datadog Agent and Cluster Agent require a token to secure their communication. So it is recommended to save this token in a `Secret` that both the Cluster Agent and Agent can reference into their Environment Variable `DD_CLUSTER_AGENT_AUTH_TOKEN`. As this will help maintain consistency and avoid the token being readable in the `PodSpec`.
```

Comment:
```suggestion
The Datadog Agent and Cluster Agent require a token to secure their communication. It is recommended that you save this token in a `Secret` that both the Datadog Agent and Cluster Agent can reference in the environment variable `DD_CLUSTER_AGENT_AUTH_TOKEN`. This helps to maintain consistency and to avoid the token being readable in the `PodSpec`.
```